util/fipstools/capture_hash/capture_hash.go

```diff
@@ -0,0 +1,66 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// capture_hash runs another executable that has been linked with libcrypto. It expects the libcrypto to run the power
+// on self tests and fail due to a fingerprint mismatch. capture_hash parses the output, takes the correct fingerprint
```

Comment:
I suggest keeping "power on" on the same line. I think it's better written "power-on self-test", but it occurs in many places, so I won't ask to change it :).

Reply:
I see "power on" changed to "power-on" in many places, but not here yet. Is that intended?